versions.lock

```diff
@@ -149,13 +149,13 @@ org.apache.ivy:ivy:2.4.0 (3 constraints: 0826dbf1)
 org.apache.orc:orc-core:1.5.6 (2 constraints: d011de27)
 org.apache.orc:orc-mapreduce:1.5.5 (1 constraints: c30cc227)
 org.apache.orc:orc-shims:1.5.6 (1 constraints: 420aebbc)
-org.apache.parquet:parquet-avro:1.10.1 (1 constraints: 35052a3b)
-org.apache.parquet:parquet-column:1.10.1 (3 constraints: 9429eeca)
-org.apache.parquet:parquet-common:1.10.1 (2 constraints: 4c1e7385)
-org.apache.parquet:parquet-encoding:1.10.1 (1 constraints: ca0ef964)
-org.apache.parquet:parquet-format:2.4.0 (3 constraints: e72a97ca)
-org.apache.parquet:parquet-hadoop:1.10.1 (2 constraints: de1ac5b3)
-org.apache.parquet:parquet-jackson:1.10.1 (1 constraints: b70ee763)
+org.apache.parquet:parquet-avro:1.11.0 (1 constraints: 35052c3b)
```

Comment:
Is it possible to separate the upgrade to Parquet 1.11.0? Or is it necessary to update both Avro and Parquet at the same time?

Reply:
Good point. Let's see in https://github.com/apache/incubator-iceberg/pull/708